cloud-director-container/package.json

```diff
@@ -25,7 +25,7 @@
     "@clr/ui": "^0.13.11",
     "@ngrx/store": "^6.0.1",
     "@vcd/bindings": "^9.1.1",
-    "@vcd/sdk": "^0.11.0",
+    "@vcd/i18n": "0.0.6",
```

Comment:
This should be whatever the current @next version is to fix timing issue on load.

Also, doesn't this remove the @vcd/sdk dependency? How is the basic testing workflow still functional?